department-of-veterans-affairs/va.gov-cms
# VAGOV-2807 records section migration

To test:
Run va_benefits_records, "Migrate Records benefits pages from VA.gov" and va_hub,
"Records landing page ..." (both in "Migrations for all new benefits pages" group)
Verify that it creates 4 benefits detail pages and 1 benefits landing page, and they all have an Administration/Owner of "Records benefits hub".
